daemon/cluster/cluster.go

```diff
@@ -1088,7 +1084,7 @@ func (c *Cluster) RemoveNode(input string, force bool) error {
 	defer c.RUnlock()
 
 	if !c.isActiveManager() {
-		return c.errNoManager()
+		return errors.NewRequestNotAcceptableError(c.errNoManager())
```

Comment:
It seems like it would make more sense to have `errNoManager()` return this `NewRequestNotAcceptableError()` error directly, since every call is being wrapped the same way.